src/Pointer.js

```diff
@@ -72,6 +72,23 @@ export class Pointer {
       this._nextTick = trigger();
     };
 
+    const _mouseLeave = (e) => {
+      if (e.target === this.config.root) {
+      Object.assign(this.previousProgress, this.currentProgress || this.progress);
```

Comment:
Why do we need this?

Reply:
Regarding this check, shouldn't be needed for `pointerleave`.
But I was actually asking about the `Object.assign(...` part. Why is it needed here?